From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Sun, 26 Mar 2017 14:30:55 +0200
Subject: Drop the patch version for breeze

Breeze doesn't even receive changes on every point release, and really lts
components shouldn't introduce incompatible changes in the same
major.minor, so dropping this unnecessary requirement.
---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,6 +3,8 @@
 project(PlasmaIntegration)
 set(PROJECT_VERSION "5.10.5")
 set(PROJECT_VERSION_MAJOR 5)
+string(REGEX MATCHALL "[0-9]+" PROJECT_VERSION_COMPONENTS ${PROJECT_VERSION})
+list(GET PROJECT_VERSION_COMPONENTS 1 PROJECT_VERSION_MINOR)
 
 include(FeatureSummary)
 find_package(ECM 5.17.0  NO_MODULE)
@@ -44,7 +46,7 @@
     PURPOSE "Required to pass style properties to native Windows on X11 Platform"
 )
 
-find_package(Breeze ${PROJECT_VERSION} CONFIG)
+find_package(Breeze ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR} CONFIG)
 set_package_properties(Breeze PROPERTIES
                        TYPE REQUIRED
                        PURPOSE "For setting the default QStyle name")
